inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -69,16 +75,106 @@ class XmlSerializer : public ngraph::AttributeVisitor {
 public:
     XmlSerializer(pugi::xml_node& data,
                   std::ostream& bin_data,
-                  std::string& node_type_name)
+                  std::string& node_type_name,
+                  const std::map<std::string, ngraph::OpSet>& custom_opsets)
         : m_xml_node(data)
         , m_bin_data(bin_data)
-        , m_node_type_name(node_type_name) {
+        , m_node_type_name(node_type_name)
+        , m_custom_opsets(custom_opsets) {
+    }
+
+    void map_type_from_body(const pugi::xml_node& xml_node,
```

Comment:
Is any reason why `output` is taking by reference? For me is cleaner when function return by value.

Reply:
force of habit, I have changed implementation and also added check for empty vectors.